test/integration/logging/local_live_log_test.js

```diff
@@ -67,8 +67,10 @@ suite('live logging', () => {
     let artifactUrl = `https:\/\/queue.taskcluster.net/v1/task/${taskId}/runs/0/artifacts/public/logs/live.log`
 
     // Don't follow redirect, we just care about where it's going
-    let res = await request.get(artifactUrl).redirects(0).end();
```

Comment:
Pretty sure `redirect(0)` still works... It's just that it now throw an error on non-2xx reponse codes..